net/raft/raft_test.go

```diff
@@ -157,6 +136,53 @@ func TestClusterSetup(t *testing.T) {
 	if got != "bar" {
 		t.Errorf("reading /foo, nodeC got %q want %q", got, "bar")
 	}
+	nodeA.cleanup()
```

Comment:
It's probably better to `defer` each of these calls just after line 127 where the nodes are created.